### ISSUE TITLE
external markdown links do not respect EXT_LINKS_IN_WINDOW

### ISSUE THREAD

**Issue by georg-emg** [doxygen/doxygen#8713]
**Describe the bug**
When using a markdown link of the form:
~~~
[Doxygen](https://www.doxygen.nl)
~~~

the resulting link is opened in the same browser window, even if `EXT_LINKS_IN_WINDOW` is set to `YES`.

**Expected behavior**
Markdown links using URLs instead of `\ref` link to external documentation and should hence respect `EXT_LINKS_IN_WINDOW`.

**To Reproduce**
[external-markdown-links-do-not-respect-EXT_LINKS_IN_WINDOW.zip](https://github.com/doxygen/doxygen/files/6973665/external-markdown-links-do-not-respect-EXT_LINKS_IN_WINDOW.zip)

**Version**
Doxygen version 1.9.1 on 64 bit Windows

**Comment by albert-github** [comment]
Looking at the markdown output (`doxygen -d markdown`) gives:
```
======== Markdown =========
---- input -------

@mainpage
[Doxygen](https://www.doxygen.nl)

---- output -----

@mainpage
<a href="https://www.doxygen.nl">Doxygen</a>

=========
```
So in the markdown processing the `EXT_LINKS_IN_WINDOW` setting is not taken in consideration as it is done at other places. most likely the function `externalLinkTarget` from util.cpp should be called during the markdown processing as well.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8719

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.2.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
